packages/mgt-components/src/components/mgt-person/mgt-person.ts

```diff
@@ -688,7 +740,9 @@ export class MgtPerson extends MgtTemplatedComponent {
                 ${initials}
               `
               : html`
-                <i class="ms-Icon ms-Icon--Contact contact-icon"></i>
+                <i class="contact-icon">
```

Comment:
Do we have a story to show this case?

Reply:
yes, person fallback details story and person vertical story both show initials text in place of image.

Reply:
This case is the icon fallback for where the initials have no length though, I can't seem to find a case where we render the personIcon svg

Reply:
Oh got it. Adding it shortly.

Reply:
Thank you for adding that